source/indexer/test/Indexer-unit.js

```diff
@@ -370,6 +370,62 @@ contract('Indexer Unit Tests', async accounts => {
       equal(stakedAmount, 350, 'Staked amount did not increase')
     })
 
+    it('should failed updating intent when tranfers tokens fail', async () => {
```

Comment:
I'm assuming these file changes exist because you branched off https://github.com/airswap/airswap-protocols/pull/300?

Reply:
yes, and will merge back once i've updated the message.